torchvision/models/segmentation/deeplabv3.py

```diff
@@ -83,7 +83,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 
 
 class ASPP(nn.Module):
-    def __init__(self, in_channels: int, atrous_rates: List[int], out_channels: int = 256) -> None:
+    def __init__(self, in_channels: int, atrous_rates: Tuple[int, int, int], out_channels: int = 256) -> None:
```

Comment:
From my understanding, using `List[int]` instead of a fixed 3-tuple was on inteded (details in https://github.com/pytorch/vision/issues/2136).


```suggestion
    def __init__(self, in_channels: int, atrous_rates: List[int], out_channels: int = 256) -> None:
```